.changeset/tricky-rockets-attend.md

```diff
@@ -0,0 +1,14 @@
+---
+"@web5/common": patch
+"@web5/crypto": patch
+"@web5/crypto-aws-kms": patch
+"@web5/dids": patch
+"@web5/credentials": patch
+"@web5/agent": patch
+"@web5/user-agent": patch
+"@web5/proxy-agent": patch
+"@web5/identity-agent": patch
+"@web5/api": patch
+---
+
+Updated all packages to use ESLint v9
```

Comment:
this probably shouldn't have a changeset should it?

Reply:
I intentionally did, because this is the last of the dependencies changes, some of which are dev dependencies, which was merged without changeset (since they came from dependabot).

Reply:
Actually, I updated the message to better reflect the intent. Good call out!

Reply:
@shamilovtim, maybe you meant what @LiranCohen meant, which is addressed now. Let me know if you meant something else need to be addressed.